system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/identityhub/IdentityHubIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.dataspaceconnector.system.tests.identityhub;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.OkHttpClient;
+import org.eclipse.dataspaceconnector.identityhub.client.IdentityHubClientImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.dataspaceconnector.system.tests.utils.TestUtils.requiredPropOrEnv;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public class IdentityHubIntegrationTest {
+
+    static final String PROVIDER_IDENTITY_HUB_URL = requiredPropOrEnv("PROVIDER_IDENTITY_HUB_URL", "http://localhost:8181/api/identity-hub");
+    static final String CONSUMER_EU_IDENTITY_HUB_URL = requiredPropOrEnv("CONSUMER_EU_IDENTITY_HUB_URL", "http://localhost:8182/api/identity-hub");
+    static final String CONSUMER_US_IDENTITY_HUB_URL = requiredPropOrEnv("CONSUMER_US_IDENTITY_HUB_URL", "http://localhost:8183/api/identity-hub");
+
+    private static final OkHttpClient OK_HTTP_CLIENT = new OkHttpClient.Builder()
+            .connectTimeout(1, TimeUnit.MINUTES)
+            .writeTimeout(1, TimeUnit.MINUTES)
+            .readTimeout(1, TimeUnit.MINUTES)
+            .build();
```

Comment:
nit: Just an idea no changes needed.

If you see this one can be reused by other tests as well somewhere then it can be moved to TestUtils.

Reply:
Good point. For now its the only one, so I suggest to do this once we have a second test using it.